src/entries/popup/hooks/useWalletsSummary.ts

```diff
@@ -61,7 +61,7 @@ const parseAddressSummary = ({
   }).amount;
   const ethCurrencyBalance = convertAmountAndPriceToNativeDisplay(
     ethBalance || 0,
-    nativeAssets?.[`${ETH_ADDRESS}_1`]?.price?.value || 0,
+    nativeAssets?.[ChainId.mainnet]?.price?.value || 0,
```

Comment:
we have other chains below following the same pattern, shouldn't we change it for all networks?